### PR TITLE
IGNITE-12918 Fix Travis config: separate Java jobs

### PR DESCRIPTION
**Pull request by ptupitsyn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,26 +23,32 @@ matrix:
...
+          --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
+          --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED
+          --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED
+          --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED
+          -Djdk.tls.client.protocols=TLSv1.2'
+      script: mvn package -Pall-java,licenses,lgpl,checkstyle,examples $JDK_PROFILES -DskipTests -Dmaven.javadoc.skip=true -B -V
+      jdk: openjdk11
+      # ignite-scalar module currently doesn't build under jdk11
+      env:
+        - JDK_PROFILES='-P!spark-2.4,!spark,!scala'
```
Comment (Mmuzaf) [inline .travis.yml:51]: I think `JDK_PROFILES` is not needed anymore since it can be directly set to an appropriate JDK profile.

Comment (ptupitsyn) [reply .travis.yml:51]: @Mmuzaf do you mean I should inline this env var, put the profiles directly to `script`?

Comment (Mmuzaf) [reply .travis.yml:51]: @ptupitsyn yes, exactly

**Comment by ptupitsyn** [comment]
Messed up the branch, had to create a new PR: #7712

**Comment by ptupitsyn** [comment]
PR autoclosed due to a mistakenly deleted branch. New one: #7713

**Pull request closed without merging**